src/services/salary/salaryService.ts

```
@@ -122,62 +125,63 @@ export async function processSalaryBatch(batchId: string): Promise<void> {
+        const result = await createTransfer({
+          senderUserId: batch.userId,
+          to: item.recipientAddress,
+          amountAcbu: item.amount.toString(),
+        });
+        await prisma.salaryItem.update({
+          where: { id: item.id },
+          data: {
+            status: result.status,
+            transactionId: result.transactionId,
+            errorMessage:
+              result.status === "failed" ? "Transfer payment failed" : null,
```

ESLint

[error] Delete `⏎·············`

(prettier/prettier)